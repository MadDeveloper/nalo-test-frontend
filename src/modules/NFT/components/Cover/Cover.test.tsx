import renderer from "react-test-renderer"
import { Cover } from "."

const nft = {
  id: "1",
  name: "Origins Sculpture",
  totalMinted: 8,
  price: "2.78",
  like: 195,
  forSale: 3,
  timeLeft: "22 hours",
  isHot: true,
  sellerId: 4321,
  image: "/assets/images/nfts/origin-sculpture.jpeg",
}

describe("NFT module > Cover", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Cover nft={nft} />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
