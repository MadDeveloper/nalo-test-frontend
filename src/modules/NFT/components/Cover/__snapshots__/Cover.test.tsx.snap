// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NFT module > Cover should match snapshot 1`] = `
<div
  className="root"
  style={
    Object {
      "backgroundImage": "url(/assets/images/nfts/origin-sculpture.jpeg)",
    }
  }
>
  <div
    className="root timeLeft"
  >
    <span
      className="root"
    >
      22 hours
       left ðŸ”¥
    </span>
  </div>
</div>
`;
